chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/domain/ChangeRequestType.java

```diff
@@ -17,4 +17,7 @@
     private Long id;
     private String name;
 
+    public Boolean isDemographic() {
+        return !this.name.equalsIgnoreCase("Developer Attestation Change Request");
```

Comment:
Should this be a constant?

Reply:
It certainly could. It seemed like overkill, as the whole file is visible on one screen, and adding a constant that's only used in one place just seemed wasteful...